pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2293,9 +2300,12 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 
 	config.DomainName = ipspec.GetDomain()
 	// Parse NTP Server
+	if config.NtpServers == nil {
+		config.NtpServers = make([]string, 0)
+	}
 	if n := ipspec.GetNtp(); n != "" {
-		config.NtpServer = net.ParseIP(n)
-		if config.NtpServer == nil {
+		config.NtpServers = append(config.NtpServers, n)
+		if config.NtpServers == nil {
```

Comment:
`config.NtpServers` will never be null here because it's being initialized right above when it's null....

Reply:
done